bindings/go/Makefile.am

```diff
@@ -5,6 +5,10 @@
 
 if HAVE_GO
 
+EXTRA_DIST = \
```

Comment:
Automake includes all found source files to distribution. However, we should include them manually in case of go. The similar was done for java bindings.